frontend/packages/noobaa-storage-plugin/src/components/capacity-card/capacity-card-body.tsx

```diff
@@ -65,21 +65,25 @@ export const CapacityCardBody: React.FC<CapacityCardBodyProps> = ({ isLoading, m
   if (isLoading) {
     return <LoadingInline />;
   }
+
   // trying to avoid redundant code
   const { metricData, metricLegend, total } = getMetricDataAndLegend(metricsData);
   if (!metricData.length) {
+    const error = total === null || total === 0;
     return (
-      <EmptyState className="chart-empty-state" variant={EmptyStateVariant.full}>
-        <EmptyStateIcon size="sm" icon={ChartPieIcon} />
-        <Title size="sm">No Prometheus datapoints found</Title>
-      </EmptyState>
+      <div className="nb-capacity-card__item">
+        <div className="co-capacity-card__item-title">Usage Breakdown</div>
+        <h6 className="co-capacity-card__item-description">Not available</h6>
+        <GaugeChart data={{ x: '', y: 0 }} label="" loading={isLoading} error={error} />
```

Comment:
stilll using `GaugeChart`, but it should be `ChartUtilization`. @yuvalgalanti can you please take a look and let us know how not available state should look like for this chart ?
Another issue I see is that the chart will jump around because if you have data, you have legend but there is non in not available state and everything is centered

Reply:
ack

Reply:
@rawagner, Does this look fine?
![Screenshot from 2019-08-09 15-05-51](https://user-images.githubusercontent.com/12200504/62771609-3f96aa00-babb-11e9-8e9f-a5ae8aca4be3.png)

Reply:
getting closer, but line is too thin and `No data` text seems too small ?

i'd imagine something like this
![breakdown](https://user-images.githubusercontent.com/2078045/62854192-dbf4c280-bcee-11e9-954c-639b5263d537.png)

```suggestion
export const CapacityCardBody: React.FC<CapacityCardBodyProps> = ({
  isLoading,
  metricsData,
  totalUsage,
}) => {
  if (isLoading) {
    return <LoadingInline />;
  }
  const { metricData, metricLegend } = getLegendAndChartData(metricsData, totalUsage);

  const error = !metricData || !metricData.length;
  const totalHumanized = humanizeBinaryBytesWithoutB(totalUsage, null, totalUsage ? null : 'Mi');
  return (
    <div className="nb-capacity-card__chart-body">
      <div className="noobaa-capacity-card__item">
        <ChartDonut
          width={275}
          height={150}
          legendData={metricLegend}
          data={error ? [{ y: 100 }] : metricData}
          labels={(datum) =>
            error ? null : `${datum.x}: ${humanizePercentage(totalUsage ? datum.y : 0).string}`
          }
          themeColor={error ? ChartThemeColor.gray : ChartThemeColor.multi}
          themeVariant={ChartThemeVariant.light}
          title={error ? 'No Data' : totalHumanized.string}
        />
      </div>
    </div>
  );
};
```

@andybraren can you take a look ?

Reply:
@rawagner what if we show the chart in center or do something with the empty space, the empty space on left doesnt looks nice IMO.
Also , showing `nodata` legend on right side, isnt cool!
What I suggest to have a graph icon similar to the ones in data consumption /area/line charts.
Since this chart does not fully follow the capacity cards of other two dashboards.
just a POV :)

Reply:
I agree, lets use `GraphEmpty` here too :)